packages/framework/ecs.php

```diff
@@ -141,6 +141,7 @@
         PropertyTypeHintSniff::class => [
             __DIR__ . '/tests/Unit/Component/ClassExtension/Source/*/*.php',
             __DIR__ . '/tests/Unit/Component/ClassExtension/Source/*.php',
+            ...explode("\n", trim(shell_exec("grep -rl '\\* @ORM\\\\Entity' " . escapeshellarg(__DIR__ . '/src')) ?? ''))
```

Comment:
I am not able to say what this does, do we really want it like that?

Reply:
I expected the comment on this line :D 

We cannot ignore `__DIR__ . '/src/*'` because individual configs are merged together when running from monorepo. So this check is then disabled completely for framework package.

The meaning of this line is: 
- find all files containing `* @ORM\Entity` in the `__DIR__ . '/src'` directory, convert the (string) output to an array, and all items from the array spread here.

I was not able to come up with a better solution.